presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/Statistics.java

```diff
@@ -352,41 +352,44 @@ private static Map<String, Map<ColumnStatisticType, Block>> createColumnToComput
     private static HiveColumnStatistics createHiveColumnStatistics(
             ConnectorSession session,
             DateTimeZone timeZone,
-            Map<ColumnStatisticType, Block> computedStatistics,
+            Map<ColumnStatisticMetadata, Block> computedStatistics,
             Type columnType,
             long rowCount)
     {
         HiveColumnStatistics.Builder result = HiveColumnStatistics.builder();
-
+        Map<ColumnStatisticType, Block> computedStatisticsTypes = computedStatistics.entrySet().stream().collect(Collectors.toMap(
+                e -> e.getKey().getStatisticType(), Entry::getValue));
         // MIN_VALUE, MAX_VALUE
         // We ask the engine to compute either both or neither
-        verify(computedStatistics.containsKey(MIN_VALUE) == computedStatistics.containsKey(MAX_VALUE));
-        if (computedStatistics.containsKey(MIN_VALUE)) {
-            setMinMax(session, timeZone, columnType, computedStatistics.get(MIN_VALUE), computedStatistics.get(MAX_VALUE), result);
+        verify(computedStatisticsTypes.containsKey(MIN_VALUE) == computedStatisticsTypes.containsKey(MAX_VALUE));
```

Comment:
I think these will always be false since `computedStatisticsTypes` is `Map<ColumnStatisticMetadata, Block>` which has a key type of `ColumnStatisticMetadata` and not `ColumnStatisticType`?